### PR TITLE
Enabling contract and token tests

### PR DESCRIPTION
**Pull request by mgoelswirlds**
**Description**:
Enabling contract and token tests in config.

This PR modifies
* Enables contracts tests for monitor.
* Enables token tests for monitor, which will run if tokenId is set in the config.

**Related issue(s)**:

Fixes #

**Notes for reviewer**:
Testing in progress on all user facing environments.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-rest/monitoring/token_tests.js [hunk 1]
```diff
@@ -533,6 +533,10 @@ const checkTokenBalanceFreshness = async (server) => {
  * @param {ServerTestResult} testResult shared server test result object capturing tests for given endpoint
  */
 const runTests = async (server, testResult) => {
+  if (!tokenIdFromConfig) {
```
Comment (mgoelswirlds) [inline hedera-mirror-rest/monitoring/token_tests.js:536]: I do get a stale token error for freshness check on tokens, for `testnet` and `mainnet` with the configured tokens. Options to resolve it:
1. Find a fresh tokenId.
2. remove that check if not necessary.

**Comment by mgoelswirlds** [comment]
I get a `stale token error` for freshness check on tokens, for `testnet` and `mainnet` with the configured tokens. Options to resolve it:
1. Find a fresh tokenId.
2. Remove that check if it is not necessary.

**Review by steven-sheehy — APPROVED** [review]

**Comment by mgoelswirlds** [comment]
The testing is complete on `previewnet`,`testnet` and `mainnet`.

**Review by jascks — APPROVED** [review]
LGTM

**Pull request merged**